task_manager/core/tasks.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
```

Comment:
I think this will greatly disrupt the current structure. As I understand it, task_manager.core is a package of interfaces. I don’t see any point in transferring the storage interface from core to storages, since this is an implementation package